ci-operator/config/amq-streams-qe/strimzi-kafka-operator/amq-streams-qe-strimzi-kafka-operator-main__amq-streams-ocp4.18-lp-interop.yaml

```diff
@@ -13,10 +13,12 @@ base_images:
     tag: release-0.43.x
 releases:
   latest:
-    candidate:
+    prerelease:
       product: ocp
-      stream: nightly
-      version: "4.18"
+      version_bounds:
+        lower: 4.17.0-0
+        stream: 4-stable
+        upper: 4.18.0-0
```

Comment:
What exactly does it mean? It sounds to me like it will install prerelease of 4.18 but not higher than 4.18.0 which doesn't makes sense to me (I didn't find much info in the docs about versions_bounds).

Reply:
@Frawless we are using this config to get the latest stable release of 4.18. This is what worked in our testing to obtain the right version

Reply:
I see, ok :)